phpunit.xml.dist

```diff
@@ -1,13 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<phpunit bootstrap="vendor/autoload.php"
+<phpunit bootstrap="./tests/bootstrap.php"
          backupGlobals="false"
          backupStaticAttributes="false"
          colors="true"
          convertErrorsToExceptions="true"
          convertNoticesToExceptions="true"
          convertWarningsToExceptions="true"
          processIsolation="false"
-         stopOnFailure="true">
+         forceCoversAnnotation="true"
```

Comment:
If you're set `forceCoversAnnotation` to `true` you need to use @covers annonation in tests.
Now coverage of code in this PR equals to 0:
![image](https://user-images.githubusercontent.com/356993/61357924-a8ab4900-a892-11e9-99c1-3b6a9785c0a3.png)

Reply:
fixed